src/sql/workbench/parts/notebook/models/cell.ts

```diff
@@ -481,7 +481,7 @@ export class CellModel implements ICellModel {
 			if (serverInfo && serverInfo.options && serverInfo.options['clusterEndpoints']) {
 				let endpoints: notebookUtils.IEndpoint[] = serverInfo.options['clusterEndpoints'];
 				if (endpoints && endpoints.length > 0) {
-					endpoint = endpoints.find(ep => ep.serviceName.toLowerCase() === 'knox');
+					endpoint = endpoints.find(ep => ep.serviceName.toLowerCase() === 'gateway');
```

Comment:
As mentioned in scrum, please support both gateway and knox (try gateway, then fall back to knox). Otherwise CTP 2.4 support would break.

Reply:
fixed.